### DISCUSSION TITLE
Can age encrypt email messages instead of just files?

### DISCUSSION THREAD

**Discussion by PowerPress** [FiloSottile/age#234] [Q&A]
Can age encrypt email messages instead of just files?

**Comment by Fastidious** [comment]
Use `age` with `-a` option on a file with the text of your email, and use the resultant file content as the email itself. Easily scripted if using `mutt` or similar email client.

**Comment by moritzdietz** [comment]
Please take a look at the spec at https://age-encryption.org/v1 and see the section "Out of scope"
> Anything about emails (which are a fundamentally unsecurable medium)

Though, you can encrypt text and put the encrypted file in an E-Mail, but this is nothing like S/MIME or GPG encrypted E-Mails.

**Reply by skpy** [reply]
> this is nothing like S/MIME or GPG encrypted E-Mails.

I've only just started looking at AGE, but can you explain how (manually) encrypting a text to send via email is different than GPG encrypted email? I feel like I'm missing something.

**Reply by moritzdietz** [reply]
I implied the whole Web of Trust thing there

**Reply by skpy** [reply]
Ah yes. That makes sense.  Thanks!
